### ISSUE TITLE
[BUG] Alltalk does not get the correct format for the voice

### ISSUE THREAD

**Issue by gunnarsal** [SillyTavern/SillyTavern#3140]
### Environment

🪟 Windows

### System

Brave Version 1.73.91, Arc 1.29

### Version

SillyTavern 1.12.8 'release' (ce34f14f1)

### Desktop Information

Node v21.5.0
Release version of ST
Using OOBABOOGA locally and TheBloke_Llama-2-13B-chat-GPTQ
Running AllTalk locally.

### Describe the problem

When I generate demo audio through AllTalk system it works fine.

When I try the same voice in ST, I get the following error
![image](https://github.com/user-attachments/assets/2a41296e-c949-4f89-9204-e54c651f2bd3)

 in AllTalk console I see the same thing

[AllTalk API] Error with API request: 1 validation error for JSONInput
narrator_voice_gen
  String should match pattern '^.*\.wav$' [type=string_pattern_mismatch, input_value='female_06', input_type=str]
    For further information visit https://errors.pydantic.dev/2.6/v/string_pattern_mismatch
    
Looks like ST is not sending over the indicator what voice to use in the correct format (?)
    Settings below
![image](https://github.com/user-attachments/assets/846b656c-a8d0-44e9-87c4-95cae32ead12)

It did work fin about 4-5 days ago

### Please tick the boxes

- [X] I have explained the issue clearly, and I included all relevant info
- [X] I have checked that this [issue hasn't already been raised](https://github.com/SillyTavern/SillyTavern/issues?q=is%3Aissue)
- [X] I have checked the [docs](https://docs.sillytavern.app/) ![important](https://img.shields.io/badge/Important!-F6094E)

**Comment by Cohee1207** [comment]
Have you tried updating AllTalk?

@erew123 you might want to take a look. Seeing that after #3060

**Comment by erew123** [comment]
@gunnarsal What version of AllTalk are you using? AllTalk V1 or AllTalk V2?

If AllTalk V1 have you recently installed it? (just wondering if the package requirements have moved up at some point as I dont really touch AllTalk V1 anymore, its kind of legacy).

Please let me know.

Thanks

**Comment by erew123** [comment]
@gunnarsal Actually I can see your problem. The Narrator has remained "Enabled" but you have No voice set for the Narrator!

I would (in this order):

- Hit the "Reload" button at the top to ensure it has pulled all the correct settings (in case you have swapped which TTS engine its using) 
- Ensure you have set the correct AllTalk server version.
- Select a Narrator voice.

That should clear it our.

**Comment by gunnarsal** [comment]
> @gunnarsal Actually I can see your problem. The Narrator has remained "Enabled" but you have No voice set for the Narrator!
> 
> I would (in this order):
> 
> * Hit the "Reload" button at the top to ensure it has pulled all the correct settings (in case you have swapped which TTS engine its using)
> * Ensure you have set the correct AllTalk server version.
> * Select a Narrator voice.
> 
> That should clear it our.

Thank you for your quick reply. I apologize if my reply is all over the place but I hope I cover everything...

I honestly do not know which version I am running. I installed it some months ago already I think. How can I check? 😥 

This narrator thing is actually another item that came up right after i posted this bug. Previously it was saying that female 06 voice was incorrect format as posted above, but now it started to mention narrator. When I try to choose narrator voice though, the dropdown is empty...
![image](https://github.com/user-attachments/assets/5a56b8ca-d5b7-40c7-8b23-a08bd248a01c)

As mentioned, voices for character/user can be choosen...

I have disabled the narrator from AT interface actually and also selected streaming option that should not have narrator (?) and had narrator set as disabled in SillyTavern as well, but it still throws that error.

![image](https://github.com/user-attachments/assets/8bfdaec7-bd67-4ec2-af3d-7ca4462e9072)
![image](https://github.com/user-attachments/assets/cb18f2eb-583b-4ee1-beaf-bff1b7ad6cd1)
![image](https://github.com/user-attachments/assets/38b0e13e-ea3b-4766-83c9-070a7438d402)

PS! Tried both V1 and V2 in the dropdown 😞
